gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -77,6 +78,7 @@ def __init__(self,
       alts: A dictionary mapping alt keys to vcf alt metadata values.
       formats: A dictionary mapping format keys to vcf format metadata values.
       contigs: A dictionary mapping contig keys to vcf contig metadata values.
+      samples: A list of sample names.
       file_name: The file name of the vcf file.
```

Comment:
As discussed offline, given that this actually encapsulates 'gs:/<bucket>/<pseudo-folder>file_name', lets rename it to file_path.